src/pages/task/taskTabs/buildBaronAndAnnotations/AnnotationTicketsTable/AnnotationTicketsTable.tsx

```diff
@@ -0,0 +1,243 @@
+                leftGlyph={<Icon glyph={isIssue ? "ArrowDown" : "ArrowUp"} />}
+              >
+                Move To {isIssue ? "Suspected Issues" : "Issues"}
+              </Button>
+            ),
+          })}
+          {ConditionalWrapper({
+            condition: userCanModify,
+            wrapper: (children: JSX.Element) => (
+              <Popconfirm
+                icon={null}
+                placement="topRight"
+                title={`Do you want to delete this ${issueString}?`}
+                onConfirm={() => {
+                  handleRemove(url, issueKey);
+                }}
+                okText="Yes"
+                cancelText="Cancel"
+              >
+                {children}
+              </Popconfirm>
+            ),
+            altWrapper: (children: JSX.Element) => (
+              <Tooltip trigger={children}>
+                You are not authorized to edit failure details
+              </Tooltip>
+            ),
+            children: (
+              <Button
+                size="small"
+                data-cy={`${issueKey}-delete-btn`}
+                leftGlyph={<Icon glyph="Trash" />}
+                disabled={!userCanModify}
+              />
+            ),
+          })}
+        </ButtonContainer>
+      ),
+    },
+  ];
+
+  const [removeAnnotation] = useMutation<
+    RemoveAnnotationIssueMutation,
+    RemoveAnnotationIssueMutationVariables
+  >(REMOVE_ANNOTATION, {
+    onCompleted: () => {
+      dispatchToast.success(`Successfully removed ${issueString}`);
+    },
+    onError(error) {
+      dispatchToast.error(
+        `There was an error removing the ${issueString}: ${error.message}`
+      );
+    },
+    refetchQueries: ["GetSuspectedIssues", "GetIssues"],
+  });
+
+  const [moveAnnotation] = useMutation<
+    MoveAnnotationIssueMutation,
+    MoveAnnotationIssueMutationVariables
+  >(MOVE_ANNOTATION, {
+    onCompleted: () => {
+      dispatchToast.success(
+        `Successfully moved ${issueString} to ${
+          isIssue ? "suspected issues" : "issues"
+        }`
+      );
+    },
+    onError(error) {
+      dispatchToast.error(
+        `There was an error moving the ${issueString}: ${error.message}`
+      );
+    },
+    refetchQueries: ["GetSuspectedIssues", "GetIssues"],
+  });
+
+  const handleRemove = (url: string, issueKey: string): void => {
+    const apiIssue = {
+      url,
+      issueKey,
+    };
+    removeAnnotation({ variables: { taskId, execution, apiIssue, isIssue } });
+    const analyticsType = isIssue
+      ? "Remove Annotation Issue"
+      : "Remove Annotation Suspected Issue";
+    annotationAnalytics.sendEvent({
+      name: analyticsType,
+    });
+  };
+
+  const handleMove = (apiIssue: {
+    url: string;
+    issueKey: string;
+    confidenceScore: number;
+  }): void => {
+    moveAnnotation({ variables: { taskId, execution, apiIssue, isIssue } });
+    const analyticsType = isIssue
+      ? "Move Annotation Issue"
```

Comment:
it could make things simpler to query if we send the analytic event name is "Move annotation" along with an "annotationType". This is what we do for mainline commits analytics

Reply:
I started doing this but then i realized the purpose of this pr is to make using task annotations easier to use and increase adoption. If we change existing analytics events we will break the tracking.